### PR TITLE
Add yarn clean command

### PR DESCRIPTION
**Pull request by grilix**
Add a shortcut command for cleaning up the environment.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "run-s": "^0.0.0"
   },
   "scripts": {
+    "clean": "rm -rf node_modules ios/build ios/Pods",
```
Comment (poocart) [inline package.json]: please also remove ios/Podfile.lock

Comment (jsidorenko) [reply package.json]: What about windows users?

Comment (grilix) [reply package.json]: @jsidorenko not sure of what we can do about Windows; we already use unix commands on this same file.

We could have two different entries, like `clean`, `clean-windows` (or something like that); or we could have a script that detects the system and do as appropriate, but I don't know if any of those is really worth it.

What do you think? Any suggestion?

Comment (jsidorenko) [reply package.json]: @grilix check that library: https://www.npmjs.com/package/rimraf

Comment (grilix) [reply package.json]: Updated

Comment (jsidorenko) [reply package.json]: 👍 And where are the changes of yarn.lock?

Comment (grilix) [reply package.json]: @jsidorenko no changes on `yarn.lock`, we already have it: https://github.com/pillarwallet/pillarwallet/blob/7e725f7785cdbabd59a1574ee907432ccf2ff437/yarn.lock#L11453

Comment (jsidorenko) [reply package.json]: Cool, we can merge it then

**Pull request merged**